extending/index.po

```diff
@@ -85,10 +110,14 @@ msgid ""
 "those tools, rather than being a recommended way to create your own C "
 "extensions."
 msgstr ""
+"Cette section du guide couvre la création d'extensions C et C++ sans "
+"l'utilisation d'outils tiers. C'est l'objectif principal des créateurs de "
```

Comment:
« C'est l'objectif principal des créateurs » ne traduit pas « It is intended primarily for creators », le sens de la phrase ici est de dire que cette section du guide est à l'intention des créateurs d'outils.